.github/workflows/java-sample-app-ecr-deploy.yml

```diff
@@ -1,6 +1,8 @@
 ## Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 ## SPDX-License-Identifier: Apache-2.0
 
+# This workflow is for building and uploading the Java sample application to ECR. Java 11 will be built and uploaded to all regions to be used by the canary while other versions (8, 17, 21, 22)
```

Comment:
nit: Line is long, should be split further

Reply:
shortened